sphinx/ext/autosummary/templates/autosummary/module.rst

```diff
@@ -34,3 +34,16 @@
    {%- endfor %}
    {% endif %}
    {% endblock %}
+
+{% block modules %}
+{% if modules %}
+.. rubric:: Modules
+
+.. autosummary::
+   :toctree: modules
```

Comment:
this argument should be removed. Then all stub-files are generated in the same directory.